### DISCUSSION TITLE
About translations in modrinth app and web

### DISCUSSION THREAD

**Discussion by shihyeon** [modrinth/.github#254] [Feature requests]
Currently the modrinth project is officially in English. We believe this project should support multiple local languages, not just English-speaking countries. (Of course, I'm not saying that English is bad.) However, I think localization is still necessary because I think it should be accessible to people who have difficulty speaking English.
(**This is the translation setting of the site itself, not a personal project translation.**)

Translation is in progress [here](https://crowdin.com/project/modrinth), but multi-language has not yet been applied. Also, theseus has not been translated. theseus is a good app and I expect it to get even better with translations.

I think you could add a language selection in the top right, for example [here](https://support.modrinth.com/en/).

We will open a discussion to see if this can be reviewed positively.

**Comment by blryface** [comment]
Afaik l10n/i18n is in the works, but isn't possible with current knossos
idk Abt Theseus tho, but there is a crowdin project for the site at https://crowdin.com/project/modrinth
